data-collection/deploy/deploy-data-collection.yaml

```diff
@@ -99,6 +102,7 @@ Mappings:
   StepFunctionCode:
     main-v1:        {TemplatePath: cfn/data-collection/source/step-functions/main-state-machine-v1.json}
     crawler-v1:     {TemplatePath: cfn/data-collection/source/step-functions/crawler-state-machine-v1.json}
+    awsfeeds:       {TemplatePath: cfn/data-collection/source/step-functions/awsfeeds-state-machine.json}
```

Comment:
Use a version number so that it will be redeployed if the template needs to change. Changing the underlying state machine template only will not  be picked up for updates to the main template.

Reply:
Done as suggested